src/macros.js

```diff
@@ -233,6 +233,11 @@ defineMacro("\u225D", "\\stackrel{\\tiny\\mathrm{def}}{=}");
 defineMacro("\u225E", "\\stackrel{\\tiny\\mathrm{m}}{=}");
 defineMacro("\u225F", "\\stackrel{\\tiny?}{=}");
 
+// Misc Unicode
+defineMacro("\u27C2", "\\perp");
+defineMacro("\u27C2", "\\mathclose{!\\mkern-0.8mu!}");
+defineMacro("\u27C2", "\\notni");
```

Comment:
All codepoints point to `27C2`

Reply:
Well that's embarrassing. @ylemkimon thank you for catching that.  Fixed now.

Reply:
And for corroboration, here's a snapshot.
![u1](https://user-images.githubusercontent.com/16403058/40634343-48f180a4-62a9-11e8-9d15-f3bc5427f129.PNG)